source/module_basis/module_nao/hydrogen_radials.cpp

```diff
@@ -0,0 +1,412 @@
+#include "module_basis/module_nao/hydrogen_radials.h"
+#include "module_base/global_variable.h"
+#include "module_base/math_integral.h"
+#include <map>
+#include <iostream>
+#include <algorithm>
+
+HydrogenRadials& HydrogenRadials::operator=(const HydrogenRadials& rhs)
+{
+    RadialSet::operator=(rhs);
+    return *this;
+}
+
+void HydrogenRadials::build(const int itype,
```

Comment:
it is the central function, pure virtual, called by RadialCollection, iteratively generate hydrogen-like orbitals, save and indiced by (l,zeta)